MyHeartCounts/Modules/StudyDefinitionLoader.swift

```diff
@@ -0,0 +1,58 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import OSLog
+import Spezi
+import SpeziStudyDefinition
+
+
+@Observable
+final class StudyDefinitionLoader: Module, Sendable {
+    enum LoadError: Error {
+        case unableToFetchFromServer(any Error)
+        case unableToDecode(any Error)
+    }
+    
+    static let studyDefinitionUrl: URL = "https://firebasestorage.googleapis.com/v0/b/myheart-counts-development.firebasestorage.app/o/public%2FmhcStudyDefinition.json?alt=media"
```

Comment:
Wondering if we can actually construct this based on the information provided by the Firebase SDK?

Reply:
the firebase sdk might not yet be configured and initialized by the time we first need to access this

Reply:
Oh, I see, good point! Can we somehow load the project identifier based on the PLIST files that we have in the setup (depending on the country they have selected)? That should at least be dynamic?

Reply:
done.